common/hdr-model.h

```diff
@@ -24,16 +24,25 @@ struct hdr_config
 
     struct preset_item
     {
-        int iterations;
+        int iterations;  // current item will be applied for this many iterations
         std::vector< control_item > controls;
 
         preset_item();
+
+        bool operator==( const preset_item & other ) const
+        {
+            if (controls[0].depth_gain != other.controls[0].depth_gain)
+                return false;
+            if (controls[0].depth_man_exp != other.controls[0].depth_man_exp)
```

Comment:
Can controls be empty when calling operator==? Then you will create it unintentionally